### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by jimpthechimp** [3kh0/ext-remover#728] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
A lot of methods don't work now because my chrome://extension ID/manifest.json page is blocked and chrome://hang stopped working. I showed the method to too many people so it was mostly my fault but what do you guys recommend?

**Comment by Su4o** [comment, marked as answer]
there's tons of new methods available. one that may work is the "google sign in exploit". Don't ask questions, but actually test out all the exploits that you think it works. You don't have to wait for other people to tell you if some works or if some doesn't.

If none of them worked. then, forget about it. A school chromebook is used for school. Schools stupid. 😞
